INSTRUCTION.md

```diff
@@ -0,0 +1,10 @@
+1. Instruction on  how to deploy 
+  ll
```

Comment:
The command 'll' is not a standard command in all environments. Consider replacing it with 'ls -l' for listing directory contents, which is more universally available.